index.css

```diff
@@ -1,68 +1,58 @@
 @import "https://fonts.googleapis.com/css?family=Lato:400,700";
 
+:root {
```

Comment:
Nie przywiązywałbym się w nazwach kolorów do miejsca ich użycia, bo np. gdy użyjesz `--font-color` w np. `background`, to nazwa nie będzie poprawna.
Nazwij te kolory kierując się ich odcieniem, dam przykład (od razu poprawiam wartościami z Figmy):
```
--white: white;
--light-grey: #f8f3f3;
--light-green: #b4e3b3;
--green: #4e924c;
```